Assets/ItemGenerator.cs

```diff
@@ -0,0 +1,40 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class ItemGenerator : MonoBehaviour {
```

Comment:
This class is a little bit convoluted.
Why does it to be like this?
Why not just create x items say every 10 seconds using Invoke Repeating.
And items themselves have DestroyByTime, so that after a while they disappear?